src/py2app/apptemplate/setup.py

```diff
@@ -20,7 +20,7 @@
     {
         "name": "main-x86_64",
         "target": "10.5",
-        "cflags": "-g -arch x86_64 -Wl,-rpath,@executable_path/../Frameworks",
+        "cflags": "-g -arch x86_64",
```

Comment:
I'm trying to work out why the issues are popping up only for apple silicon... As far as I know, this is the only difference. If it has no change at runtime, maybe it has an effect at build-time? This could be a quick lead which could be easy to test experimentally.

Reply:
I sent a probe in #507 and #508. I personally got a negative result on arm64 (#508), so it's likely that #315 truly did do nothing... and this massive bug has existed for more than 3 years?

Reply:
> I sent a probe in #507 and #508. I personally got a negative result on arm64 (#508), so it's likely that #315 truly did do nothing... and this massive bug has existed for more than 3 years?

yep,Even now, still can't pack torchaudio. It's unbelievable